### ISSUE TITLE
WSL2 guides yield "requested resource NotFound 404"

### ISSUE THREAD

**Issue by basraven** [krustlet/krustlet#730]
When following [the KinD Guide](https://docs.krustlet.dev/howto/krustlet-on-kind/),
I'm getting:
```
Error: ApiError: the server could not find the requested resource: NotFound (ErrorResponse { status: "Failure", message: "the server could not find the requested resource", reason: "NotFound", code: 404 })

Caused by:
    the server could not find the requested resource: NotFound
```
The same occurs when following [the WSL2 Guide](https://docs.krustlet.dev/howto/krustlet-on-wsl2/).

_(Tested on different machines with Windows 10 & 11)_

I used the following steps:
1. Get Krustlet (also tested with v1.0.0-alpha.1)
    ```bash
    wget https://krustlet.blob.core.windows.net/releases/krustlet-v0.7.0-linux-amd64.tar.gz
    tar -xzf krustlet-v0.7.0-linux-amd64.tar.gz
    ```

2. Create cluster
    ```bash
    kind create # Or enable the Kubernetes cluster in docker-desktop on Windows
    ```
3. Run the bootstrap.sh script (seems to be working fine)
    ```bash
     bash <(curl https://raw.githubusercontent.com/krustlet/krustlet/main/scripts/bootstrap.sh)
    ```
4. Install and run Krustlet
    ```bash
    export mainIP=$(ifconfig eth0 | grep "inet " | awk '{ print $2 }')

    KUBECONFIG=~/.krustlet/config/kubeconfig \
    ./krustlet-wasi \
    --node-ip $mainIP \
    --node-name=krustlet \
    --bootstrap-file=${HOME}/.krustlet/config/bootstrap.conf
    ```

I tried a bunch of other IP addresses in a combination of changing it in --node-ip and in ~/.krustlet/config/bootstrap.conf
- 127.0.01
- host ip (192.168.5.1)
- WSL ip of host
- WSL ip
- Any random ip such as 8.32.56.134

All give this same error, meaning it just cannot find the API server it seems

**Comment by Falumpaset** [comment]
Hey, did you resolve this issue? I'm encountering the same problem.

**Comment by basraven** [comment]
> Hey, did you resolve this issue? I'm encountering the same problem. Probably the Kubernetes Api has changed since the release. Testing on Kubernetes 1.23 and Kubectl 1.25

Nope, I'm still having the same issue unfortunately, tried several things in the mean time, but nothing worked

**Comment by CapitanMurloc** [comment]
The same thing happens to me, I wasn't sure if I was doing something wrong. But if there are more people then surely there is a problem in the documentation or in the versioning as they say.

**Comment by nt-7** [comment]
Can you try canary-builds? I resolved by it.
[krustlet - canary builds](https://docs.krustlet.dev/intro/install/#from-canary-builds)
